gems/aws-sdk-core/lib/aws-sdk-core/process_credentials.rb

```diff
@@ -73,9 +73,9 @@ def refresh
       @credentials = credentials_from_process(@process)
     end
 
-    def near_expiration?
-      # are we within 5 minutes of expiration?
-      @expiration && (Time.now.to_i + 5 * 60) > @expiration.to_i
-    end
+    # def near_expiration?
```

Comment:
I'm not 100% sure this code should be here. This class inherits `RefreshingCredentials`, but when I'm inspecting it -- I'm not sure if it's actually doing what y'all want(?) And in any regards, I think it's a valid refactor as well, but figured I'd pose it as an open question rather than a suggestion (hence the commented out code).

Reply:
@alextwoods 🏓 on this if you didn't see. I wasn't sure if we wanted this here -- it seemed a little odd and wanted to see if I could follow boy scout's rule for y'all. If not -- can revert this change as well :)

Reply:
The `ProcessCredentials` are a bit weird - this method is actually subtly different because ProcessCredentials may not have an expiration and if so, shouldn't be refreshed at all (the default `RefreshingCredentials#near_expiration?` will return true if `@expiration` is not set.

Reply:
Yeah I noticed this too. I can revert the change then, I just wanted to make sure -- thanks for your explanation 🙂